RELEASE_NOTES.md

```diff
@@ -1,7 +1,15 @@
-# Cosmos SDK v0.45.10 Release Notes
+# Cosmos SDK v0.45.11 Release Notes
 
-This release introduces a number of bug fixes, features and improvements.
+This release introduces a number of bug fixes and improvements.
 
 Please see the [CHANGELOG](https://github.com/cosmos/cosmos-sdk/blob/release/v0.45.x/CHANGELOG.md) for an exhaustive list of changes.
 
-**Full Commit History**: https://github.com/cosmos/cosmos-sdk/compare/v0.45.9...v0.45.10
+**Full Commit History**: https://github.com/cosmos/cosmos-sdk/compare/v0.45.10...v0.45.11
+
+**NOTE**: The changes mentioned in `v0.45.9` are **still** required:
+
+> Chains must add the following to their go.mod for the application:
+>
+> ```go
+> replace github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
+> ```
```

Comment:
```suggestion
```go
# Chains must add the following to their go.mod for the application:
replace github.com/confio/ics23/go => github.com/cosmos/cosmos-sdk/ics23/go v0.8.0
```
```

Reply:
Why? We did it like that in 0.46 too. It looks nice: https://github.com/cosmos/cosmos-sdk/releases/tag/v0.46.4

Reply:
Because, it doesn't make sense to inline a code block to me personally.

Reply:
Alright, updated here and in the other releases notes on GitHub.